milvus/grpc/Data.ts

```diff
@@ -460,7 +460,7 @@ export class Data extends Collection {
 
         scores.splice(0, topk).forEach((score, scoreIndex) => {
           // get correct index
-          const i = index === 0 ? scoreIndex : scoreIndex + topk;
+          const i = index === 0 ? scoreIndex : scoreIndex + topk * index;
```

Comment:
array.splice will change the scores array.length. I don't think we need to multiple index here.

Reply:
There is a bug in the previous version. the search results should return multiple array if you are searching with multiple vectors. but it is fixed in the main branch, will be released tomorrow I think.

Reply:
thanks